src/Backend/OnboardingInsuranceAPI/Areas/User/GetUser.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.EntityFrameworkCore;
+using OnboardingInsuranceAPI.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OnboardingInsuranceAPI.Areas.User;
+
+public class GetUser : IHandler
+{
+    private readonly DataContext _context;
+    public GetUser(DataContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<UserData> GetUserBy(string pid)
+    {
+        //await _context.Database.EnsureDeletedAsync(); //uncomment it in case DB is empty
+        //await _context.Database.EnsureCreatedAsync();
```

Comment:
I wanted to ask how to implement it creation fake db, because i have override OnModelCreating method on  Data context so I decided why not to create db based on this information ...

Reply:
Ok